review-me/review-me.js

```diff
@@ -0,0 +1,26 @@
+// hello.js
+
+var message = "Hello, world!"
+console.log(MESSAGE);
+
+function printmessage(msg) {
+    if(msg != null){
+        console.log(msg)
+    }
+    else{
+        console.log("Nothing to print")
+    }
+}
+
+printmessage(message)
```

Comment:
15  :  Function  name  ' Do Stuff '  does  not  follow  Java Script  camel Case  naming  convention  naming  incons istency .